quickstarts/ocs-install-tour-quickstart.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: ocs-install-tour
```

Comment:
This is missing the required cluster profile annotations.

```
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"```

Reply:
added. thanks